python/tk_desktop/rpc.py

```diff
@@ -443,17 +464,17 @@ def call(self, name, *args, **kwargs):
                     if self._closed:
                         raise RuntimeError("client closed while waiting for a response")
                     continue
-            except IOError:
-                # An exception is raised on Windows when the connection
-                # is closed during polling instead of simply returning False.
-                if self._closed:
-                    raise RuntimeError("client closed while waiting for a response")
-                raise
             except OSError as e:
+                # IOError it's an alias of OSError
                 if self._closed:
-                    raise RuntimeError(
-                        "client closed while waiting for a response with OSError exception"
-                    )
+                    raise RuntimeError("client closed while waiting for a response")
+
+                # Check if there's a WinError that causes some flaky behavior on CI
+                if sys.platform == "win32" and "The handle is invalid" in str(e):
```

Comment:
I prefer that the string "The handle is invalid" be a variable, for example a class variable, so:
`if sys.platform == self.WIN32 and self._INVALID_ HANDLE_MESSAGE in str(e):`

same for "win32" because it is used plenty times in this class